lua/terrortown/lang/en.lua

```diff
@@ -2227,6 +2227,19 @@ L.help_locational_voice_range = [[This convar constrains the maximum range at wh
 
 Set to 0 to disable this cut-off.]]
 
+-- 2024-04-07
+L.help_voice_activation = [[Changes the way your microphone is activated for global voice chat. These all use your "Global Voice Chat" keybinding. Team voice chat is always push-to-talk.
```

Comment:
I think we use single `'` for in line strings. I'm not 100% sure though. Maybe you could check that